collectors/StatsCollector.py

```diff
@@ -37,30 +37,28 @@ def collect(self):
             return
 
         values_received = set()
+        no_match_in_config = []
 
         for resource in values:
             resource_id = resource.get('resourceId')
             labels = self.set_labels(resource_id, project_ids)
 
             for value_entry in resource.get('stat-list', {}).get('stat', []):
                 statkey = value_entry.get('statKey', {}).get('key')
-                values_received.add(statkey)
+                norm_statkey = re.sub("[^a-zA-Z|_ ]+", "", statkey)
```

Comment:
Normalising keys here. Sometimes we get renamed metrics from API like `cpu:102|usage_average` instead of `cpu|usage_average`. This won't match the given statkey and producing way to much noisy metrics.

Reply:
please add this as a comment